pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/JsonConverter.java

```diff
@@ -197,12 +197,7 @@ JsonNode toJson(Schema schema, Object value) {
                 new Conversions.UUIDConversion()) {
             @Override
             JsonNode toJson(Schema schema, Object value) {
-                if (!(value instanceof String)) {
-                    throw new IllegalArgumentException("Invalid type for uuid, expected String but was "
-                            + value.getClass());
-                }
-                String uuidString = (String) value;
-                return jsonNodeFactory.textNode(uuidString);
+                return jsonNodeFactory.textNode(value.toString());
```

Comment:
can we do a null check here ?
crashing with an NPE is very hard to debug

Reply:
Ok, added a check for null